### PR TITLE
Feature/registration page

### PR DESCRIPTION
**Pull request by nikitaprodanov**
Created a simple registration page to store users into database.
Has validations for already taken username so no duplicates.

### CODE REVIEW

**Review by velisav-boyanov — APPROVED** [review]
След щателна проверка мисля, че всичко е функционално.

File: wtform_fields.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	confirm_pswd = PasswordField('confirm_pswd_label', 
+		validators=[InputRequired(message="Password confirmation required"), 
+		EqualTo('password', message="Passwords must match")])
+	submit_button = SubmitField('Create')
+
+	# Custom validator for the username
+	def validate_username(self, username):
+		user_object = User.query.filter_by(username=username.data).first();
+		if user_object:
+			raise ValidationError("Username already exists. Please choose another username")
```
Comment (shristov-elsys) [inline wtform_fields.py:24]: This is a security issue. Never return messages as ""Username already exists" or "Wrong password!". This gives the end user ability to hack your software. If this user is already existing then we can confirm that  certain username exists.  for example "shristov". Then  we could try username shristov with brute force attack or even better with phishing. That's why it is returned always "Wrong username or password!" and nothing more or less. Look at google or other big platforms

**Review by shristov-elsys — CHANGES_REQUESTED** [review]

File: wtform_fields.py (hunk 1 above)
Comment (uchihanaruto9371) [reply wtform_fields.py]: This is only the registration page so your password really can`t be incorrect. All big platforms at the stage of registration are informing you that the username is already taken.
For example (gmail.com):

taken-username

Comment (nikitaprodanov) [reply wtform_fields.py:24]: This is only the registration page so your password really can`t be incorrect. All big platforms at the stage of registration are informing you that the username is already taken.
For example (gmail.com):

already-taken

**Review by shristov-elsys — APPROVED** [review]

**Pull request merged**